src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -141,6 +142,7 @@ def load_arguments(self, _):
         c.argument('assign_kubelet_identity', type=str, validator=validate_assign_kubelet_identity)
         c.argument('disable_local_accounts', action='store_true')
         c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
+        c.argument('workload_runtime', arg_type=get_enum_type([CONST_WORKLOAD_RUNTIME_OCI_CONTAINER, CONST_WORKLOAD_RUNTIME_WASM_WASI]))
```

Comment:
I suppose it would be better if you could create a global variable such as `workload_runtimes = [CONST_WORKLOAD_RUNTIME_OCI_CONTAINER, CONST_WORKLOAD_RUNTIME_WASM_WASI]`, then set this variable to `get_enum_type`, which would avoid forgetting to modify the optional parameters of create or update in the future.

Reply:
From help message, I suppose we should add `default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER`?

Reply:
better?